### ISSUE TITLE
Error caused by op 'model/optimizer/gradients/model/loss/strided_slice_grad/StridedSliceGrad'

### ISSUE THREAD

**Issue by youssefsharief** [keithito/tacotron#196]
I am currently training for arabic words

The training data looks like this

> nawar-spec-00001.npy|nawar-mel-00001.npy|1223| وَرَجَّحَ التَّقْرِيرُ الَّذِي أَعَدَّهُ مَعْهَدُ أَبْحَاثِ هَضَبَةِ التِّبِتِ فِي الْأَكَادِيمِيَّةِ الصِّينِيَّةِ لِلْعُلُومِ - أَنْ تَسْتَمِرَّ دَرَجَاتُ الْحَرَارَةِ وَمُسْتَوَيَاتُ الرُّطُوبَةِ فِي الْإِرْتِفَاعِ طَوَالَ هَذَا الْقَرْنْ

At the beginning I was getting the error below 
> Exiting due to exception: Incompatible shapes: [32,1295,80] vs. [32,1000,80]
This error was resolved by increasing the value of "max_iters"
Then I am currently stuck with this error below

> [[Node: model/optimizer/gradients/model/inference/decoder/while/BasicDecoderStep/decoder/output_projection_wrapper/output_projection_wrapper/multi_rnn_cell/cell_1/gru_cell/BiasAdd_1/Enter_grad/b_acc_3/_115 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_1400_...ad/b_acc_3", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]
Hint: If you want to see a list of allocated tensors when OOM happens, add report_tensor_allocations_upon_oom to RunOptions for current allocation info.

> Caused by op 'model/optimizer/gradients/model/loss/strided_slice_grad/StridedSliceGrad', defined at:
>   File ".\train.py", line 152, in <module>
>     main()
>   File ".\train.py", line 148, in main
>     train(log_dir, args)
>   File ".\train.py", line 65, in train
>     model.add_optimizer(global_step)
>   File "C:\Code\Repos\tacotron\tacotron_1\models\tacotron.py", line 136, in add_optimizer
>     gradients, variables = zip(*optimizer.compute_gradients(self.loss))
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\training\optimizer.py", line 514, in compute_gradients
>     colocate_gradients_with_ops=colocate_gradients_with_ops)
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\ops\gradients_impl.py", line 596, in gradients
>     gate_gradients, aggregation_method, stop_gradients)
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\ops\gradients_impl.py", line 779, in _GradientsHelper
>     lambda: grad_fn(op, *out_grads))
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\ops\gradients_impl.py", line 398, in _MaybeCompile
>     return grad_fn()  # Exit early
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\ops\gradients_impl.py", line 779, in <lambda>
>     lambda: grad_fn(op, *out_grads))
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\ops\array_grad.py", line 278, in _StridedSliceGrad
>     shrink_axis_mask=op.get_attr("shrink_axis_mask")), None, None, None
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\ops\gen_array_ops.py", line 10387, in strided_slice_grad
>     shrink_axis_mask=shrink_axis_mask, name=name)
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\framework\op_def_library.py", line 787, in _apply_op_helper
>     op_def=op_def)
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\util\deprecation.py", line 454, in new_func
>     return func(*args, **kwargs)
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\framework\ops.py", line 3155, in create_op
>     op_def=op_def)
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\framework\ops.py", line 1717, in __init__
>     self._traceback = tf_stack.extract_stack()
> 
> ...which was originally created as op 'model/loss/strided_slice', defined at:
>   File ".\train.py", line 152, in <module>
>     main()
> [elided 0 identical lines from previous traceback]
>   File ".\train.py", line 148, in main
>     train(log_dir, args)
>   File ".\train.py", line 64, in train
>     model.add_loss()
>   File "C:\Code\Repos\tacotron\tacotron_1\models\tacotron.py", line 119, in add_loss
>     self.linear_loss = 0.5 * tf.reduce_mean(l1) + 0.5 * tf.reduce_mean(l1[:,:,0:n_priority_freq])
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\ops\array_ops.py", line 524, in _slice_helper
>     name=name)
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\ops\array_ops.py", line 690, in strided_slice
>     shrink_axis_mask=shrink_axis_mask)
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\ops\gen_array_ops.py", line 10187, in strided_slice
>     name=name)
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\framework\op_def_library.py", line 787, in _apply_op_helper
>     op_def=op_def)
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\util\deprecation.py", line 454, in new_func
>     return func(*args, **kwargs)
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\framework\ops.py", line 3155, in create_op
>     op_def=op_def)
>   File "C:\Users\User\Anaconda3\lib\site-packages\tensorflow\python\framework\ops.py", line 1717, in __init__
>     self._traceback = tf_stack.extract_stack()
